openlimits/src/coinbase/mod.rs

```diff
@@ -86,22 +86,34 @@ impl Exchange for Coinbase {
     }
 
     async fn get_all_open_orders(&self) -> Result<Vec<Order<Self::OrderIdType>>> {
-        coinbase::Coinbase::get_all_open_orders(self)
+        let params = GetOrderRequest {
+            status: Some(String::from("open")),
+            paginator: None,
+            product_id: None,
+        };
+
+        coinbase::Coinbase::get_orders(self, Some(&params))
             .await
             .map(|v| v.into_iter().map(Into::into).collect())
     }
 
     async fn get_order_history(
         &self,
-        req: &crate::model::GetOrderHistoryRequest,
+        req: &GetOrderHistoryRequest,
     ) -> Result<Vec<Order<Self::OrderIdType>>> {
-        coinbase::Coinbase::get_all_orders(self, req.symbol.as_deref())
+        let req: GetOrderRequest = req.into();
+
+        coinbase::Coinbase::get_orders(self, Some(&req))
             .await
             .map(|v| v.into_iter().map(Into::into).collect())
     }
 
-    async fn get_account_balances(&self) -> Result<Vec<Balance>> {
-        coinbase::Coinbase::get_account(self)
+    async fn get_account_balances(&self, paginator: Option<&Paginator>) -> Result<Vec<Balance>> {
+        let paginator: Option<coinbase::model::Paginator> = paginator.map(|p| p.into());
+
+        // Transform a Option<T> to Option<&T>
```

Comment:
`my_option.as_ref()` can convert Option<T> to Option<&T>. the other way around, `my_ref_option.clone()` will transform Option<&T> to Option<T> assuming T implements Clone

Reply:
I'm not sure exactly what you need this for just looking at the code since this function returns `Vec<Balance>`, but responding here given your slack question